components/example-notebook-servers/jupyter-pytorch-full/requirements.txt

```diff
@@ -1,6 +1,5 @@
 # kubeflow packages
-kfp==2.4.0
-kfp-server-api==2.0.3
+kfp==2.7.0
```

Comment:
@kubeflow/wg-notebooks-leads @kubeflow/wg-training-leads Can we install [`kubeflow-training`](https://pypi.org/project/kubeflow-training/) and [`kubeflow-katib`](https://pypi.org/project/kubeflow-katib/) Python SDK as part of Kubeflow Notebooks images ?
Similar to KFP ?

Reply:
Is there a particular version of these packages that aligns with the 1.9 release?

Reply:
Could we include the `kubeflow-training==1.8.0rc0` and `kubeflow-katib==0.17.0rc0` ?
We haven't released the 1.8 and 0.17 version yet.

Reply:
@andreyvelich lets wait until the final versions of those packages are released.

Can you try and cut them soon, so we can include them in the next RC?

Adding packages will require me to run all the tests again for all the images, which I don't have time to do before we cut the first RC for 1.9.0.